api/apps/api/src/modules/scenarios/scenarios.controller.ts

```diff
@@ -92,8 +83,8 @@ export class ScenariosController {
   async findAll(
     @ProcessFetchSpecification() fetchSpecification: FetchSpecification,
   ): Promise<ScenarioResult> {
-    const results = await this.service.findAllPaginated(fetchSpecification, {});
-    return this.service.serialize(results.data, results.metadata);
+    const results = await this.service.findAll(fetchSpecification);
+    return this.scenarioSerializer.serialize(results.data, results.metadata);
```

Comment:
we should keep pagination here

Reply:
```
async findAll(fetchSpecification: FetchSpecification) {
    return this.crudService.findAllPaginated(fetchSpecification);
  }
```
we do 🤔 

Will change tho.

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/241/commits/8e4e9a68d948000c3dda981102d5afb8a224884d